cast/js/rhino/src/main/java/com/ibm/wala/cast/js/translator/TypedNodeVisitor.java

```diff
@@ -158,6 +161,12 @@ public R visit(AstNode node, A arg) {
       return visitSwitchCase((SwitchCase) node, arg);
     } else if (node instanceof SwitchStatement) {
       return visitSwitchStatement((SwitchStatement) node, arg);
+    } else if (node instanceof TaggedTemplateLiteral) {
+      return visitTaggedTemplateLiteral((TaggedTemplateLiteral) node, arg);
```

Comment:
See comment above about not visiting these nodes if we don't yet handle them; or is the issue that we'll get an unexpected node type?

In general, what happens right now when we try to pass an ES6 input to WALA, given that we haven't set the language version?  Do we just get a parse error in Rhino?